INSTALL.md

```diff
@@ -0,0 +1,101 @@
+# From new CentOS 7 install:
+
+## Disable selinux:
+### change `/etc/selinux/config` to `SELINUX=disabled`
+
+## Install Postgreql 9.6 client and libraries
+
+Instructions are here: https://yum.postgresql.org/
+
+- grab the link for CentOS 7 and install:
+    $ sudo yum install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-xxxx.noarch.rpm
```

Comment:
@dangogh Since the url is not valid, It may be good to add a note "where xxxx is a integer, such as, 3 or latest version" . I tried running that command at first to realize that you wanted me to supply a valid value for xxxx.